dkan.profile

```diff
@@ -411,3 +411,109 @@ function dkan_set_roleassign_roles(&$context) {
   }
   variable_set('roleassign_roles', $roleassign_roles);
 }
+
+/**
+ * Configures BUEditor and markdown test format.
+ */
+function dkan_bueditor_markdown_install() {
```

Comment:
Ideally it would get called upon install as well as updates. As we add these functions I would think they would live in the dkan.profile.